src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => {/* Fill me in */ }}>
+                <button onClick={() => {this.props.increment(this.props.count)}}>
```

Comment:
You actually don't need to pass in `this.props.count`, for 2 reasons:
1: You are not using your payload in your actions, and when you pass in `this.props.count` that's where it is going.
2: `this.props.count` is actually your total number after incrementing/decrtementing. Console log it, and you will see what i mean. And it is coming from, `mapStateToProps` at the bottom of the page
 it would do increment anyways